cluster/gce/config-default.sh

```diff
@@ -73,11 +75,16 @@ fi
 # Also please update corresponding image for node e2e at:
 # https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/jenkins/image-config.yaml
 CVM_VERSION=${CVM_VERSION:-container-vm-v20170627}
-GCI_VERSION=${KUBE_GCI_VERSION:-cos-stable-59-9460-64-0}
+# Kubemark_changes: Changed kubemark master image and project.
+#GCI_VERSION=${KUBE_GCI_VERSION:-cos-stable-59-9460-64-0}
```

Comment:
Same here - if you want to use ubuntu - you can always overwrite. Please revert.